### ISSUE TITLE
Full instance not loaded, shows "Still loading", no menu interface working

### ISSUE THREAD

**Issue by anndix** [ronivay/XenOrchestraInstallerUpdater#140]
**Debian 10**: I tried with Debian 11 first
**v16.16.0**: node -v
**1.22.19**: yarn -v

**Server specs**
16vCPU
4GB RAM

**Issue**
After install, the interface loads up with only 4 left menu options "Home", "Dashboard", "XOA", "Tasks". None of the menu options work, clicking on any option doesn't refresh or update any info on screen. Screenshot attached.

The info on right side keeps showing "Still loading"

**Installation logfile**
Attach complete log as a file. 
[xo-install.log-202208140252.log](https://github.com/ronivay/XenOrchestraInstallerUpdater/files/9332207/xo-install.log-202208140252.log)

Any help would be highly appreciated.
![Screenshot 2022-08-14 at 3 06 26 AM](https://user-images.githubusercontent.com/50706506/184511531-546fbf15-4bc6-4d9c-a9ef-9a04c48c5785.png)

**Comment by ronivay** [comment]
Hi,

can’t spot any issues from installation logs. do you see any errors in xo-server log when refreshing GUI? `journalctl -u xo-server -n -100` for example.

**Comment by anndix** [comment]
I found a strange solution. Both on Chrome and Safari show the same screen. However when i open the interface on firefox it works. I have tried to remove cache and do all combinations to try and solve, but the issue remains on chrome and safari. I have stock XOA free version VM as well, it opens properly on the same browsers.

[quote]
journalctl -u xo-server -n -100
[/quote]

The above command throws the below error:
[quote]
journalctl: invalid option -- '1'
[/quote]

Thanks

**Comment by ronivay** [comment]
> `journalctl -u xo-server -n -100`
> 
> 
> 
> The above command throws the below error:
> 
> 
> 
> `journalctl: invalid option -- '1'`
> 

Sry my typo, just 100 without the `-`

**Comment by anndix** [comment]
No error, just below 2 lines in logs. 

`Aug 15 09:53:40 mgm xo-server[509]: 2022-08-15T04:23:40.995Z xo:main INFO - Console proxy (admin - ::ffff:223.233.64.152)
Aug 15 14:47:49 mgm xo-server[509]: 2022-08-15T09:17:49.964Z xo:main INFO + Console proxy (admin - ::ffff:223.233.64.152)`

Thanks for your help, appreciate it.

**Comment by ronivay** [comment]
Hi,

Are you still experiencing this issue? I'm unable to reproduce it on firefox, chrome or safari. I see that you used the new ACME feature for Let's Encrypt certificate. Could you try without it to see if it's related. Anyway if this still happens, you should try to look into chrome/safari developer console for hints why parts of the page aren't loading properly.

**Comment by 3rdcycle** [comment]
Same issue here. Works fine in Firefox but not in Safari / Chrome. The web console shows 

`WebSocket network error: OSStatus Error -9807: Invalid certificate chain `

This is a test instance and I'm using a self generated certificate.

**Comment by anndix** [comment]
Issue persists. Unable to load it in Safari and Chrome. Only works in Firefox.

**Comment by anndix** [comment]
Hi,

Is there a way I can generate a letsencrypt certificate and install myself ? Maybe that will remove the error ?

Thanks

Anand Gupta

**Comment by ronivay** [comment]
> Hi,
> 
> Is there a way I can generate a letsencrypt certificate and install myself ? Maybe that will remove the error ?
> 
> Thanks
> 
> Anand Gupta

Sure. This repo wiki has instructions how certificates are configured. LE certificate you can get with eq. certbot which has it's own documentation available.

**Comment by ronivay** [comment]
sorry for long delay on this. i haven't found enough time to debug this issue. any help from the community is highly appreciated.

**Comment by anndix** [comment]
> sorry for long delay on this. i haven't found enough time to debug this issue. any help from the community is highly appreciated.

Hi,

If you can help guide which log files to check while loading interface i would want to try it out. If the logs aren't enabled by default then maybe we can enable logging and then debug what exactly happens when the respective browser loads the interface. 

Thanks

**Comment by ronivay** [comment]
> > sorry for long delay on this. i haven't found enough time to debug this issue. any help from the community is highly appreciated.
> 
> 
> 
> Hi,
> 
> 
> 
> If you can help guide which log files to check while loading interface i would want to try it out. If the logs aren't enabled by default then maybe we can enable logging and then debug what exactly happens when the respective browser loads the interface. 
> 
> 
> 
> Thanks


I have a feeling this isn't server side issue so there isn't most likely anything in XO logs. Browser developer tools would be the place to look for if it has some hints why parts of the content aren't loading.

**Comment by ruinsteve** [comment]
I just had this same issue but I was able to fix it by clearing my cache. I know others have tired this but it worked for me.

Background: I was using an older deployment of this running on Debian 10. The upgrade script was failing as Node was too old. So I just downloaded the latest Debian 11 based image (thanks for making these!) and set it up with the exact same settings as the old one. It was working fine at first until I uploaded the config from the old install. Then I got this exact same issue right away. Opening it in incognito worked fine. So I cleared my cache and was good to go. The only other difference is I did run the update script and reboot (both for good measure) before trying incognito.
